oclint-scripts/oclintscripts/cmake.py

```diff
@@ -36,10 +36,6 @@ def release_build(self):
     def test_build(self):
         return self.append('TEST_BUILD', '1')
 
-    def use_clang_compiler(self, llvm_root = path.build.clang_install_dir):
```

Comment:
This method name might be confusing, we are okay to rename it to `use_local_clang_compiler`.